### PR TITLE
Chapter1/homework/1 and2

### PR DESCRIPTION
**Pull request by RoshanSun**
Nice

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -6,7 +6,30 @@ class Program
...
+				float height = (float)Double.Parse(Console.ReadLine());
+
+				Console.WriteLine(firstName + lastName + " is " + age + " years old, his weight is "
+					+ weight + " kg and his height is " + height + " cm.");
+
+				float bodyMassIndex = weight / ((height / 100) * (height / 100));
+				Console.WriteLine("The BMI of " + firstName + lastName + " is " + bodyMassIndex);
+
+				Console.WriteLine("Would you like to try again, for a different person?");
+				response = Console.ReadLine();
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this useless assignment to local variable 'response'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845326)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Program.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845326&bid=16082964)

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -6,7 +6,30 @@ class Program
...
+			} while (response = "Y");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Remove this useless assignment to local variable 'response'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845326)

Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Extract the assignment of 'response' from this expression.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845326)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -6,7 +6,30 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
-        }
+			String response;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Replace all tab characters in this file by sequences of white-spaces.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845326)

**Comment by RoshanSun** [comment]
> Note the difference of String and string though.

**Comment by RoshanSun** [comment]
Woops, I misclicked something. But yeah, I come from Java so it's a bit weird.

File: Src/BootCamp.Chapter/Program.cs [hunk 4]
```diff
@@ -6,7 +6,30 @@ class Program
...
+			do
+			{
+				Console.WriteLine("Input your first name: ");
+				string firstName = Console.ReadLine();
+				Console.WriteLine("Input your surname: ");
+				string lastName = Console.ReadLine();
+				Console.WriteLine("Input your age: ");
+				int age = Int32.Parse(Console.ReadLine());
+				Console.WriteLine("Input your weight (in kg): ");
+				float weight = (float)Double.Parse(Console.ReadLine());
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: use float.Parse directly.

**Comment by ellmaster** [comment]
> Second person is also missing, please add.

I disagree. The do while loop should be fine. @Almantask

**Comment by Almantask** [comment]
Tests failing, please fix.

**Comment by Almantask** [comment]
Expired

**Pull request closed without merging**